shell/platform/darwin/macos/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -354,6 +389,35 @@ - (void)updateEditState {
   [self updateTextAndSelection];
 }
 
+- (void)updateEditStateWithDelta:(const flutter::TextEditingDelta)delta {
+  NSUInteger selectionBase = _activeModel->selection().base();
+  NSUInteger selectionExtent = _activeModel->selection().extent();
+  int composingBase = _activeModel->composing() ? _activeModel->composing_range().base() : -1;
+  int composingExtent = _activeModel->composing() ? _activeModel->composing_range().extent() : -1;
+
+  NSString* const textAffinity = [self textAffinityString];
+
+  NSDictionary* deltaToFramework = @{
+    @"oldText" : @(delta.oldText().c_str()),
```

Comment:
Does this work? `c_str()` returns a UTF-8 string but the `NSDictionary` literal should only take `NSString` I think?

Reply:
Oh the `@()` will be turned into `-[NSString stringWithUTF8String]` by the compiler.